docs/adr/2022-08-02_020-handling-time.md

```diff
@@ -0,0 +1,63 @@
+---
+slug: 20
+title: |
+  20. Handling time
+authors: []
+tags: [Proposed]
+---
+
+## Status
+
+Proposed
+
+## Context
+
+* The Hydra Head protocol is expected to be isomorphic to the ledger it runs on. That means, it should support the same transaction formats and (if desired) use the same ledger rules as the layer 1.
+
+* Cardano is our layer 1 and its consensus layer separates time into discrete steps, where each step is called a `Slot`. The network is expected to evolve strictly monotonically on this time scale and so slot numbers (`SlotNo`) are always increasing.
+
+* The Cardano mainnet has a block scheduled every 20 seconds, although it may take longer.
+  - This is because it has `f = 0.02` and slot length is one second.
```

Comment:
I see, my comment is perhaps not relevant. I would intuitively have framed things the other way round as we start introducing the concept of `Slot` first.

Reply:
But this is even wrong.. it should say 0.05, so thanks for pointing out 😅

Reply:
😂